README.md

```diff
@@ -61,6 +61,7 @@ Container environments:
 - [Colima](https://github.com/abiosoft/colima): Docker (and Kubernetes) on macOS with minimal setup
 - [Finch](https://github.com/runfinch/finch): Finch is a command line client for local container development
 - [Podman Desktop](https://podman-desktop.io/): Podman Desktop GUI has a plug-in for Lima virtual machines
+- [Meridian](https://github.com/aoxn/meridian): Conformance Kubernetes and docker container environment management on desktop based on lima
```

Comment:
To the files derived from Lima, could you add the "Copyright The Lima Authors." header.

- https://github.com/lima-vm/lima/pull/3293